src/Application/User/Command/Mail/SendForgotPasswordMailCommandHandler.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\User\Command\Mail;
+
+use App\Application\CommandBusInterface;
+use App\Application\MailerInterface;
+use App\Application\StringUtilsInterface;
+use App\Application\User\Command\CreateTokenCommand;
+use App\Domain\Mail;
+use App\Domain\User\Enum\TokenTypeEnum;
+use App\Domain\User\Exception\UserNotFoundException;
+
+final readonly class SendForgotPasswordMailCommandHandler
+{
+    public function __construct(
+        private CommandBusInterface $commandBus,
+        private MailerInterface $mailer,
+        private StringUtilsInterface $stringUtils,
+    ) {
+    }
+
+    public function __invoke(SendForgotPasswordMailCommand $command): void
+    {
+        $email = $this->stringUtils->normalizeEmail($command->email);
+
+        try {
+            $token = $this->commandBus->handle(
+                new CreateTokenCommand(
+                    $email,
+                    TokenTypeEnum::FORGOT_PASSWORD->value,
+                ),
+            );
+
+            $this->mailer->send(
+                new Mail(
+                    address: $email,
+                    subject: 'forgot_password.subjet',
+                    template: 'email/user/forgot-password.html.twig',
```

Comment:
Je viens de voir mais par suivi de la convention, utiliser le tiret du bas pour le nom du template ?

```suggestion
                    template: 'email/user/forgot_password.html.twig',
```

Reply:
@mmarchois J'ai l'impression que le dernier force push a défait des changements que tu avais fait précédemment :confused:

Reply:
J'ai le seum ...